src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -1376,12 +1400,10 @@ public boolean handleTokenExpiration(HttpServletRequest httpRequest, HttpServlet
     }
 
     private void redirectToLoginUrl(HttpServletRequest req, HttpServletResponse res) throws IOException {
-        if (req != null && (req.getSession(false) != null || Strings.isNullOrEmpty(req.getHeader("Authorization")))) {
+        if (req.getSession(false) != null || Strings.isNullOrEmpty(req.getHeader("Authorization"))) {
             req.getSession().invalidate();
         }
-        if (res != null) {
-            res.sendRedirect(Jenkins.get().getSecurityRealm().getLoginUrl());
-        }
+        res.sendRedirect(Jenkins.get().getSecurityRealm().getLoginUrl());
```

Comment:
I added the null check on purpose (see https://github.com/jenkinsci/oic-auth-plugin/releases/tag/4.438.v6e62f6782770). 

Suggestion: add `@NonNull` to the parameters to tighten the contract.